evasquez26/angular4-startup
# S2 one 61 define course lesson

## Story / Task

[ONE-61](https://bixlabs.atlassian.net/browse/ONE-61)

## What this PR does?

Creating and editing lessons for a course.

### Before 

We could not add and edit lessons for a course.

### After

We can add lessons to a course.

### How to test / reproduce

- Go to a course previously created
- Observe that there's a lessons tab
- Clicking the add new lesson button should show a new form with the data of a lesson
- Clicking the add lesson button should create the lesson and list it (this is not done yet)

### TODOs
- [x] List correctly lessons from a course, right now the only way to list lessons from a course is to add first a lesson and then create a class, and after that the lesson should reflect. But it should not work like that, instead the lesson should be listed immediately after creating it
- [x] Implement correctly the markdown editor
- [x] Rebase after merging #31 

### Related PRs

N/A

## Migrations

NO
